https/README.md

```diff
@@ -0,0 +1,24 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP.
+When running a server with TLS use the flag `--web.tls-config`
+
+e.g. `./node_exporter --web.tls-config="https/tls-config.yml"`
+If the config is kept within the https directory.
+
+The config file should be written in YAML format, and is reloaded on each connection to check for new certificates and/or authentication policy.
+
+##Sample Config:
+```
+tlsConfig :
+  # Certificate and key files for server to use to authenticate to client
+  tlsCertPath : <filename>
+  tlsKeyPath : <filename>
+
+  # Server policy for client authentication. Maps to ClientAuth Policies
+  # For more detail on clientAuth options: [ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType)
+  [ clientAuth : <string> | default = "NoClientCert" ]
+
+  # CA certificate for client certificate authentication to the server
+  [ clientCAs : <filename> ]
```

Comment:
Should we warn if the user provides this without a clientAuth setting to use it?

Reply:
Makes sense.

Reply:
The changes currently made will output a warning when this occurs. 
Is this correct or should it change the policy to RequireAndVerify and warn that because a clientCA has been added it assumes it needs that policy but the option to not have one or have a less restrictive one is still there?

Reply:
I'd error out, as it's likely a mistake by the user.